packages/create-wave-app/__tests__/copy-template.test.ts

```diff
@@ -33,15 +33,15 @@ describe("copyTemplate", () => {
     const consoleSpy = vi.spyOn(console, "error");
     consoleSpy.mockImplementationOnce(vi.fn(() => void 0));
 
-    const actual = await copyTemplate("pnpm", {
+    const actual = await copyTemplate("yarn", {
```

Comment:
what was the thinking behind switching this from pnpm to yarn? since we aren't supporting yarn anymore?

Reply:
before it was testing for pnpm, which wasn't configured so this test is meant to fail. since pnpm is now an option, i switched the test to yarn in order to make it fail